_plugins/contributors.rb

```diff
@@ -106,9 +106,42 @@ def site_payload
         return
       end
 
-      # Populate site.corecontributors and site.sitecontributors arrays
-      site.corecontributors = contributors('bitcoin/bitcoin',site.config['aliases'])
-      site.sitecontributors = contributors('bitcoin-dot-org/bitcoin.org',site.config['aliases'])
+      ## Create cache directory if it doesn't exist
+      system 'mkdir', '-p', '_cache'
```

Comment:
@harding How about the ruby equivalent `Dir.mkdir('_cache') if !File.exists?('_cache')`
(This line of code also occurs elsewhere)